R/which_element.R

```diff
@@ -0,0 +1,42 @@
+#' which_element
+#'
+#' This function returns locations within an EML list object that contain an instance where \code{element == value}
+#' @param eml an EML list object
```

Comment:
may be more clear to use the variable name, eml_list, since "eml" means the entire eml code in other cases